source/application/AMDatamanAppController.cpp

```diff
@@ -750,11 +759,11 @@ bool AMDatamanAppController::startupRegisterDatabases()
 	success &= AMDbObjectSupport::s()->registerClass<AMXESScan>();
 	success &= AMDbObjectSupport::s()->registerClass<AM2DScan>();
 	success &= AMDbObjectSupport::s()->registerClass<AM3DScan>();
-	success &= AMDbObjectSupport::s()->registerClass<AMXRFScan>();
```

Comment:
Haha yeah, I was just making sure I understood how you'd been able to remove this class. I got the answer to this too when I'd read the full pull request.